BC-MAY-21-ROR/kata-07-reloj-checador-team-01
# Set Heroku Deployment Configuration

## What does this change?
We set all the things required for a heroku deployment and then we add a set of html elements to view on the production url. Also we add a tailwindcss library for making styles. Header and footer created on partials files.

## Why are you changing that?
We made this change to have a graphical view for verifying something is displaying on production., in order to validate that Heroku was configured correctly. 

## Merge/Deploy Checklist
* [ ]   It has been reviewed and accepted.

## How do you manually test this?
You can go to Heroku production url: 
[Check-in Heroku Production App](https://still-dusk-94334.herokuapp.com/)


## Screenshots (if apply)
<img width="986" alt="Captura de Pantalla 2021-10-01 a la(s) 11 15 32" src="https://user-images.githubusercontent.com/48681779/135653797-6e6f2c84-0267-4271-ac73-49099e8e1c78.png">
